packages/foreman/rubygem-unicode/rubygem-unicode.spec

```diff
@@ -1,72 +1,94 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name unicode
 
-Name:           %{?scl_prefix}rubygem-%{gem_name}
-Version:        0.4.4.1
-Release:        5%{?dist}
-Summary:        Unicode normalization library for Ruby
-License:        Ruby
-URL:            http://www.yoshidam.net/Ruby.html#unicode
-Source0:        https://rubygems.org/gems/%{gem_name}-%{version}.gem
-# https://github.com/blackwinter/unicode/issues/7
-Source1:        https://www.ruby-lang.org/en/about/license.txt
-# This is a C extension linked against MRI, it's not compatible with other
-# interpreters. So we require MRI specifically instead of ruby(release).
```

Comment:
given this comment, should the (build)requires on `%{scl}-runtime` be kept below?